modules/allowances/hardhat.config.ts

```diff
@@ -61,10 +61,6 @@ const config: HardhatUserConfig = {
       ...sharedNetworkConfig,
       url: `https://rpc.energyweb.org`,
     },
-    goerli: {
-      ...sharedNetworkConfig,
-      url: `https://goerli.infura.io/v3/${INFURA_KEY}`,
-    },
     mumbai: {
       ...sharedNetworkConfig,
       url: `https://polygon-mumbai.infura.io/v3/${INFURA_KEY}`,
```

Comment:
This is also [deprecated](https://docs.infura.io/api/networks/polygon-pos/choose-a-network), right? It's now Amoy.